libs/feeds_processing/src/utils.rs

```diff
@@ -47,7 +48,7 @@ pub fn check_signature(
 #[derive(Debug)]
 pub struct ConsumedReports {
     pub is_quorum_reached: bool,
-    pub skip_publishing: bool,
+    pub skip_publishing: SkipDecision,
```

Comment:
Suggestion, rename `skip_publishing` to `skip_decision`

Reply:
done